app/assets/javascripts/controllers/home_controller.js.coffee

```
@@ -1 +1,6 @@
-YJ.HomeController = Ember.Controller.extend()
\ No newline at end of file
+YJ.HomeController = Ember.Controller.extend(
+
+  viewTerms: (->
+    @transitionTo('terms')
```

You don't need to handle this at the controller.  You can let the event bubble up to the home route and catch it there.  I'm going to merge this and add a follow on pull request with letting the event go to the home route.